ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/store/dao/events/GroupContiguousSpec.scala

```diff
@@ -29,36 +27,24 @@ final class GroupContiguousSpec
       }
   }
 
-  it should "be equivalent to grouping on inputs with a contiguous key" in forAll(contiguous) {
-    pairsWithContiguousKeys =>
-      val grouped = groupContiguous(Source(pairsWithContiguousKeys))(by = _._1)
-      whenReady(grouped.runWith(Sink.seq[Vector[(Int, String)]])) {
-        _ should contain theSameElementsAs pairsWithContiguousKeys.groupBy(_._1).values
-      }
+  it should "be equivalent to grouping on inputs with a contiguous key" in {
+    val pairsWithContiguousKeys = List(1 -> "baz", 0 -> "foo", 0 -> "bar", 0 -> "quux")
```

Comment:
Why only one test cases ?

Reply:
This is just to confirm that sorting is not important, the previous one covers enough cases.